src/libraries/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -676,6 +676,25 @@ public static unsafe string Title
         }
 
         public static void Beep()
+        {
+            const char BellCharacter = '\u0007'; // Windows doesn't use terminfo, so the codepoint is hardcoded.
+
+            if (!Console.IsOutputRedirected)
+            {
+                Console.Out.Write(BellCharacter);
+                return;
+            }
+
+            if (!Console.IsErrorRedirected)
```

Comment:
The Unix version of this function does not write to stderr. Should we try to be consistent between Windows and Unix - produce the bell character for `!Console.IsOutputRedirected` only?